gitarro.rb

```diff
@@ -21,14 +25,18 @@
   # this check the last commit state, catch for review or not reviewd status.
   comm_st = b.client.status(b.repo, pr.head.sha)
 
+  # force test without any filter or check
+  next if b.force_run_test(pr)
+
   # retrigger if magic word found
   b.retrigger_check(pr)
   # 0) do test for unreviewed pr
-  break if b.unreviewed_new_pr?(pr, comm_st)
+  next if b.unreviewed_new_pr?(pr, comm_st)
+
```

Comment:
Why are you using next instead of break?

Reply:
Oh yes, I didn't explained that.
I don't saw the reason why we cut the loop and not iterate through all the pending PRs of the list, running the tests in the same execution.
It make more sense than run the script n times, one for each PR with pending tests.
What do you think?

Reply:
We are doing this because:

1 Jenkins JOB = 1 test run.

If we run all the tests at same time, we will break this rule and we don't have 1 Build URL pro JOB/tests. SO we can't append the URL to the result of a single PR in GitHub.
We will run all the tests in a single Jenkins job with all the LOGs togheter, and run all the tests in same execution has other disadvantages. You have 30 min execution and imagine the worker dies for some reason. It is better to have small operation which are independent from each others.

It is better here to don't break the logic because we will add to much unknown for a PR which is sensed to remove and add

Reply:
OK, I'll revert it.
Is true that mixing PR reports is bad.